helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -1360,12 +1361,38 @@ networkExplorer:
 #
 # The OpenTelemetry Operator running as a deployment with a replica count of 1.
 # It auto-instruments applications to emit telemetry data.
-# Related documentation: docs/auto-instrumentation-install.md
+# Related documentation: https://github.com/signalfx/splunk-otel-collector-chart/blob/main/docs/auto-instrumentation-install.md
 # Full list of Helm value configurations: https://artifacthub.io/packages/helm/opentelemetry-helm/opentelemetry-operator?modal=values
 ################################################################################
 
 operator:
   enabled: false
+  # For more details, refer to: https://github.com/open-telemetry/opentelemetry-operator/blob/main/docs/api.md#instrumentation
+  instrumentation:
+    create: true
```

Comment:
Why do we need to expose this option? Isn't the instrumentation the primary goal of introducing the operator?

Reply:
This can be removed for now and added back later if we ever get a request for it. I could see a user using other methods for deploying and managing Instrumentation objects and not wanting to use our default. Not sure how common it would be though.

Removed.